lonabot/bot.py

```diff
@@ -399,6 +408,45 @@ async def tz(self, update):
                                text=f"Got it! There's a difference of "
                                     f"{delta} seconds between you and I :D")
 
+    @dumbot.command
+    async def later(self, update):
+        sender_id = update.message.from_.id
+        current_later = self.db.get_later(sender_id)
+        later = update.message.text.split(maxsplit=1)
+
+        if len(later) == 1:
+            if current_later is None:
+                await self.sendMessage(
+                    chat_id=update.message.chat.id,
+                    text='You have no "later" configured, configure it by '
+                        'using the command as such: '
+                        '/later 1y2mo3w4d5h6s'
+                )
+            else:
+                await self.sendMessage(
+                    chat_id=update.message.chat.id,
+                    text='Your current "later" delay is '
+                        f'{current_later} seconds'
```

Comment:
Do we not have a function to show a friendlier value here?

Reply:
Oh we do, `utils.spell_delay`, neat.

Reply:
These now use the `spell_delay` function.